### ISSUE TITLE
- ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code.

### ISSUE THREAD

**Issue by christinasund** [CocoaPods/CocoaPods#10203]
* [x] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

🌈

# Report

## What did you do?

ℹ Please replace these two lines with what you did.  
`pod lib lint --verbose --no-clean`
`pod spec lint --verbose --no-clean`

## What did you expect to happen?

ℹ Please replace these two lines with what you expected to happen.  
Validate the podspec

## What happened instead?

ℹ Please replace these two lines with of what happened instead.  
``` 
Testing with `xcodebuild`.
 -> TealiumCrashModule (2.1.0)
    - WARN  | github_sources: Github repositories should end in `.git`.
    - WARN  | description: The description is equal to the summary.
    - WARN  | [iOS] license: Unable to find a license file
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code.
    - NOTE  | [TealiumCrashModule/Crash] xcodebuild:  note: Using new build system
    - NOTE  | [TealiumCrashModule/Crash] xcodebuild:  note: Building targets in parallel
    - NOTE  | [TealiumCrashModule/Crash] xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | [TealiumCrashModule/Crash] xcodebuild:  note: Planning build
    - NOTE  | [TealiumCrashModule/Crash] xcodebuild:  note: Constructing build description
    - NOTE  | [iOS] xcodebuild:  clang: error: linker command failed with exit code 1 (use -v to see invocation)
    - NOTE  | [TealiumCrashModule/Crash] xcodebuild:  warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.1.99. (in target 'TealiumCrashReporter' from project 'Pods')
    - NOTE  | [TealiumCrashModule/Crash] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')

Analyzed 1 podspec.

[!] The spec did not pass validation, due to 1 error and 3 warnings..
```

I opened App.xcworkspace and the build succeeded. However, when I tried to run the App target, I get a message about a missing CFBundleString. There is no .plist for me to try and fix that. 

## CocoaPods Environment

ℹ Please replace these two lines with the output of `pod env`.
```
   CocoaPods : 1.10.0
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
    RubyGems : 3.0.3
        Host : Mac OS X 10.15.7 (19H15)
       Xcode : 12.1 (12A7403)
         Git : git version 2.24.3 (Apple Git-128)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : trunk - CDN - https://cdn.cocoapods.org/
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.4.1
cocoapods-try         : 1.1.0
```

## Project that demonstrates the issue

ℹ Please link to a project we can download that reproduces the issue.
You can delete this section if your issue is unrelated to build problems,

The podspec is here:
https://github.com/Tealium/tealium-swift-crash-reporter

But I have since added the following to get around the arm64 error:
  ```
    ...
    s.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64'}
    s.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64'}

    s.subspec "Crash" do |crash|
        ....
        crash.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64'}
        crash.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64'}
    end
```
Thank you!!

**Comment by dnkoutso** [comment]
Yes this has been covered before in another issue and so far this is the correct way to handle this.

**Comment by ahmedAlmasri** [comment]
@dnkoutso I'm trying to add `target_xcconfig ` but still `pod lib lint` get 
`xcodebuild: Returned an unsuccessful exit code.`, 

when trying to build `App.xcworkspace` manually it's working fine and build successfully
